dbt_project.yml

```diff
@@ -1,12 +1,14 @@
 config-version: 2
 name: 'github'
-version: '0.7.0'
+version: '0.8.0'
 require-dbt-version: [">=1.3.0", "<2.0.0"]
 models:
   github:
-    materialized: table
+    +materialized: table
+    +schema: github
     intermediate:
-      materialized: ephemeral
+      +materialized: ephemeral
+      +schema: int_github
```

Comment:
If these are all ephemeral then they won't actually be materialized in a schema. This is probably not necessary and can be removed.

Reply:
If these models are materialized then we are not documenting this in the README section.

Reply:
True. I kept this in case the user changed the materialization, however, if they're doing that they could also define their own schema and yeah this probably isn't necessary. Removed.